src/jitlog/build.lua

```diff
@@ -0,0 +1,46 @@
+local stdout = io.stdout
+local args = {...}
+
+if args[1] == "GC64" then
+  GC64 = true
+end
+
+--Work around limited api in when run under minilua
+if not require then
+  function print(...)
+    local t = {...}
+    for i, v in ipairs(t) do
+      if type(v) == "boolean" then
+        stdout:write((v and "true") or "false")
+      else
+        stdout:write(v)
+      end
+    end
+    stdout:write("\n")
+  end
+
+  function require(name)
+    local path = name..".lua"
+    
+    if string.find(name, "%.") then
+      local package, name = string.match(name, "([^%.]+)%.(.+)")
+      
+      if not package or not name then
+          error("bad lua module name")
+      end
+      path = package .. "/".. name..".lua"
+    end
+  
+    local fp = assert(io.open(path))
+    local s = fp:read("*a")
+    assert(fp:close())
+    return assert(loadstring(s, "@"..name..".lua"))()
+  end
+end
+
+local msgdef = require"jitlog.messages"
+local apigen = require"jitlog.generator"
+local parser = apigen.create_parser()
+parser:parse_msglist(msgdef)
+
+local data = parser:complete()
```

Comment:
Why assign to a variable that's not then used?

Reply:
Well when the file was chomped up into multiple commits I was reverting back to final target target file each time I was building it back up in commits. Its used in later commit at least

Reply:
OK. It would be good to fix this before merging though.